fe/src/components/layouts/Footer/Footer.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  Box,
+  Container,
+  Stack,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+
+export default function Footer(): JSX.Element {
+  return (
+    <Box
+      bg={useColorModeValue('gray.50', 'gray.900')}
+      color={useColorModeValue('gray.700', 'gray.200')}
+    >
+      <Container
+        as={Stack}
+        maxW="8xl"
+        py={4}
+        direction={{ base: 'column', md: 'row' }}
+        spacing={4}
+        justify={{ base: 'center', md: 'space-between' }}
+        align={{ base: 'center', md: 'center' }}
+      >
+        <Text>© 2021 Tallinn Polytechnic School. All rights reserved</Text>
```

Comment:
maybe add link to TPT website (gotta promote the school somehow)

Reply:
Sure thing.